cpp/tests/t/io/PointCloudIO.cpp

```diff
@@ -214,6 +214,56 @@ TEST(TPointCloudIO, ReadPointCloudFromPLY3) {
     EXPECT_FALSE(pcd.HasPointAttr("intensity"));
 }
 
+// Reading ascii and check for 3DGS attributes.
+TEST(TPointCloudIO, ReadPointCloudFromPLY4) {
+    std::string filename_out = utility::filesystem::GetTempDirectoryPath() +
+                               "/test_sample_right_3dgs_format.ply";
+    std::ofstream outfile;
+    outfile.open(filename_out);
```

Comment:
We can make this a global variable (anon namespace) since this is used in multiple places.

Reply:
Also, you can use raw string literals to avoid typing the \n:
```c++
R"(ply
format ascii 1.0
...)"
```